src/router/constants/authRoutes.ts

```diff
@@ -52,6 +52,10 @@ export const authRoutes = {
     route: 'my-cooperations/:id/quiz/:quizId'
   },
   accountMenu: {
+    admin: {
+      myProfile: { route: 'my-profile', path: '/my-profile' },
+      logout: { route: 'logout', path: '/logout' }
+    },
```

Comment:
why these changes appeared in this PR? looks like they are in relevant and should be moved

Reply:
im not sure someone pushed these changes , this needs for logging in as admin